From 4fb083dde2acf760a65a0b243a52e87c43dcb58b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timoth=C3=A9e=20Ravier?= <tim@siosm.fr>
Date: Mon, 5 Sep 2022 21:18:21 +0200
Subject: [PATCH] ksandbox: Use host-spawn

---
 src/lib/util/ksandbox.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/util/ksandbox.cpp b/src/lib/util/ksandbox.cpp
index 0bd01037..01bef4f6 100644
--- a/src/lib/util/ksandbox.cpp
+++ b/src/lib/util/ksandbox.cpp
@@ -46,7 +46,7 @@ KSandbox::ProcessContext KSandbox::makeHostContext(const QProcess &process)
         args << process.program();
     }
     args += process.arguments();
-    return {QStringLiteral("/usr/bin/flatpak-spawn"), args};
+    return {QStringLiteral("/usr/bin/host-spawn"), args};
 }
 
 KCOREADDONS_EXPORT void KSandbox::startHostProcess(QProcess &process, QProcess::OpenMode mode)
-- 
2.37.3

